### PR TITLE
Java menu, Add mobile-menu using JavaScript.

### PR DESCRIPTION
**Pull request by TaiwoEnoch**
Add JavaScript functions to make my mobile menu responsive, I do not do it alone, I will like to acknowledge @MuhammadUsmanAslam my coding partner, for the collaboration.

### CODE REVIEW

**Comment by huseyinbiyik** [comment]
Hi @TaiwoEnoch and @MuhammadUsmanAslam,

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

Alt Text

## Invalid Code Review Request

⚠️ You have submitted a project, that has no Stylint checks pass. Kindly fix your linter errors before submitting for review. 
Screenshot 2023-01-31 at 2 13 04 AM


Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **does not count** into the code reviews limit._

**Comment by TaiwoEnoch** [comment]
Oh, i will work on it now

**Comment by TaiwoEnoch** [comment]
Have make some corrections now, it is very okay now @huseyinbiyik

File: index.html [hunk 1]
```diff
@@ -10,19 +10,30 @@
...
+        </ul>
+          <img src="./images/Union.png"  id="toggle-button" class="mobile">
+      </nav>
+      <div id="mobile-menu" class="mobile-menu-hide">
+        <i id="mobile-menu-icon" class="fa-solid fa-x flex-end"></i>
+        <ul id="mobile-menu-links" class="mobile-menu-link">
+          <li><a href="#my-work">Portfolio</a></li>
+          <li><a href="#about">About</a></li>
+          <li><a href="#contact">Contact</a></li>
+        </ul>
```
Comment (osamaashraf6) [inline index.html:34]: - [ ] Kindly 😃try to decrease the margin between elements to stick to the Figma Design and with that the mobile menu will be attractive for the visitors and clients who will communicate with you 😊
- [ ] Also consider adding the color of the nav links of the mobile menu like the color of the Figma Design 💯

> Check below to see the difference 😊⤵️

| Figma design | Your design| 
| --- | --- |
|   image|   image |

File: index.html [hunk 2]
```diff
@@ -10,19 +10,30 @@
...
+        <ul id="navbar-links" class="nav-bar">
+          <li><a href="#my-work">Portfolio</a></li>
+          <li><a href="#about">About</a></li>
+          <li><a href="#contact">contact</a></li>
+          <li><img src="./images/Icon-Mail.png" class="mail"></li>
+        </ul>
+          <img src="./images/Union.png"  id="toggle-button" class="mobile">
+      </nav>
+      <div id="mobile-menu" class="mobile-menu-hide">
+        <i id="mobile-menu-icon" class="fa-solid fa-x flex-end"></i>
```
Comment (osamaashraf6) [inline index.html:29]: - [ ] Good job at implementing the functionality of closing the mobile menu, but Kindly 😊try to add the correct color of the ```X ``` icon to be like the Figma Design 😃
- [ ] Also consider😃 decreasing the margin between the ```X``` icon and the nav links to be like the Figma Design👌


> Check below to see the difference 😊⤵️

| Figma design | Your design| 
| --- | --- |
|   image|   image|

- [ ] I see that when I click on the hamburger icon and the mobile menu navs and the ``` X ``` appears to me and try to resize the page, the ```X``` icon and the navs is still existing in the desktop version, so we don't need this happen in the desktop we want when resize at the desktop version the mobile menu and the ```X``` icon disappear😊, so please try to fix this to achieve the project requirements 😊

> Check below to see 😊⤵️

| Your design| 
| --- |
|   image   |

**Review by osamaashraf6 — CHANGES_REQUESTED** [review]
Hi @TaiwoEnoch 👋, 

Good job so far! 👏🥳
You have done well reaching this point of the milestone, but there are some issues that you still need to work on to go to the next project but you are almost there! 

you-are-near

## Highlights ✔️

- [x] Good commit messages.
- [x] No linter errors. 💯
- [x] Good usage in correcting GitHub flow
- [x] Nice design.




## Required Changes ♻️

- [ ] I see 😃 that in the future features, you put the link of the live demo there, and inside the section of the live demo you write 
``` This link is not available now.``` , so kindly try to move the link from the future features into the live demo section to make your portfolio available for the visitors and clients who will communicate with you 😊
- [ ] check the comments under the review.

## Optional suggestions
Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better.

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by amejid — APPROVED** [review]
Hi team, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



## Highlights
- Linter checks passing :heavy_check_mark: 
- Mobile menu working as expected :heavy_check_mark: 
## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**